umu/umu_consts.py

```diff
@@ -16,26 +16,14 @@
     "getnativepath",
 }
 
-# Installation path of the runtime files
+# Installation path of the runtime files.
 # Flatpak will be detected as outlined by systemd
 # See https://systemd.io/CONTAINER_INTERFACE
-UMU_LOCAL: Path = (
-    Path.home().joinpath(
-        ".var", "app", "org.openwinecomponents.umu.umu-launcher", "data", "umu"
-    )
-    if os.environ.get("container") == "flatpak"  # noqa: SIM112
-    else Path.home().joinpath(".local", "share", "umu")
-)
+UMU_LOCAL: Path = Path.home().joinpath(".local", "share", "umu")
```

Comment:
This should prefer `HOST_XDG_DATA_HOME` and then fallback to the default.  
Flatpak manifests then should add permission to `xdg-data/umu:create`